### ISSUE TITLE
Resolve inconsistency issues in automated tests

### ISSUE THREAD

**Issue by ccanash** [w3c/aria-at-app#1231]
During stress testing of Automated test runs we found the foloowing inconsistencies:

- https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20240926-0600.consistency-report#most-common-responses-7 - weird :obj: result for NVDA Firefox on command-button
- https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20240926-0600.consistency-report#divergent-responses-from-run-1 - might be nice to try to reproduce why this one is inconsistent.... tests/alert is pretty stable on everything, but one run on voiceover safari failed to capture the "Hello" - this could be a easier to reproduce window into the timing of other "missing utterance" problems if someone could get it to happen locally with super-extra-debuggery timing logging
- https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20240926-0600.consistency-report#most-common-responses-12 - command-button voiceover chrome has "Run Test Setup" as the most common response for command 3 in the test - might point to a specific key press that is shaky - or this is just a buggy test setup. We could search the whole doc for "Run Test Setup" and try to figure out which commands are triggering it, maybe it's consistent, maybe not
- https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20240926-0600.consistency-report#most-common-responses-34 disclosure-navigation seems to have problems with response 2 as it's "most common" response even on NVDA (which is much more stable in general) - I feel like there are problems with this test, or some of it's features across the board
- https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20240926-0600.consistency-report#divergent-responses-from-run-1-23 - it is odd that we got into the system menu

**Comment by ChrisC** [comment]
@ccanash I just compared results from the sept Stress Test run and the [most recent run](https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20241213-Consistency-Report) and will respond to items listed in this issue in separate comments!

**Comment by ChrisC** [comment]
For the first item, the [most common responses](https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20241213-Consistency-Report#testscommand-button-nvda-firefox) for command-button on Firefox + NVDA now look good and consistent, so I don't think there's any weird ":obj" result anymore.

**Comment by ChrisC** [comment]
For the second item on the alert test in Safari + VO, [all `tests/alert` are running consistently now,](https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20241213-Consistency-Report#summary-by-all) so this doesn't appear to be an issue any longer either.

**Comment by ChrisC** [comment]
For the third item, [`tests/command-button` on Chrome + VO is now running consistently ](https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20241213-Consistency-Report#most-common-responses-11) without the "Run Test Setup" output, so this also no longer an issue.

**Comment by ChrisC** [comment]
For the fourth item, the [disclosure-navigation tests continue to be the flakiest](https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20241213-Consistency-Report#summary-by-test-plan). I'll take a closer and report possible issues there separately.

**Comment by ChrisC** [comment]
On the last item about getting into the MacOS system menu from `tests/menu-button-actions-active-descendant` on Firefox + VO, it looks like the [latest stress test results got truncated around here](https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20241213-Consistency-Report#testsmenu-button-actions-active-descendant-voiceover-firefox) (cc @outofambit). So we'll need to do another stress test run, to verify if this still happening. I'll file this as a separate issue.

**Comment by ChrisC** [comment]
@ccanash I'm going to follow up with separate issues from the latest stress test run, but I think you can close this issue now!

**Comment by ccanash** [comment]
Closing, we will create new tasks with the latest test run results
